prboom2/src/g_game.c

```diff
@@ -700,7 +700,15 @@ void G_BuildTiccmd(ticcmd_t* cmd)
     forward += mousey;
   }
   if (strafe)
+  {
     side += mousex / movement_mousestrafedivisor; /* mead  Don't want to strafe as fast as turns.*/
+
+    // You can only produce even strafe values in vanilla doom
+    if (demo_compatibility)
+    {
+      side &= ~1;
```

Comment:
Is this right for negative side values? 🤔 
I guess that would actually turn -1 into -2? :^)